### DISCUSSION TITLE
Is there any way to remove (env) text from start of prompt?

### DISCUSSION THREAD

**Discussion by DestroyerAce** [fish-shell/fish-shell#9430] [Q&A] [answered]
Hi I use a theme from oh-my-fish and everytime I use virtual environment it makes the theme look weird 
so is there any way to remove it    
![image](https://i.imgur.com/jKAQWpA.png)
(ignore the second (env) its from theme files :P)

**Comment by faho** [comment, marked as answer]
The `env` is your venv's name. Python's [activate.fish script](https://github.com/python/cpython/blob/main/Lib/venv/scripts/posix/activate.fish) displays it by default unless $VIRTUAL_ENV_DISABLE_PROMPT is set to non-empty.

So:

```fish
set -g VIRTUAL_ENV_DISABLE_PROMPT 1
```

And if your theme includes its own venv display it should be setting that variable.

**Reply by faho** [reply]
See e.g. how our disco prompt does it:

https://github.com/fish-shell/fish-shell/blob/f15e5ce1da53ede461dd4afc3caefae84894ec4a/share/tools/web_config/sample_prompts/disco.fish#L84-L90

**Comment by DestroyerAce** [comment]
Thanks adding that to theme worked
